charts/install_smcp_with_helm/control-plane/templates/servicemeshcontrolplane-basic-install.yaml

```diff
@@ -58,11 +58,19 @@ spec:
               targetPort: 15443
           type: ClusterIP
     egress:
+{{- if .Vaules.smcp_enable_custom_egress_config }}
```

Comment:
typo in Values

Reply:
well, at least I know copy/paste works...

fixed in second commit.